test/test_YoutubeDL.py

```diff
@@ -778,8 +778,8 @@ def expect_same_infodict(out):
         test('%(title5)#U', 'a\u0301e\u0301i\u0301 𝐀')
         test('%(title5)+U', 'áéí A')
         test('%(title5)+#U', 'a\u0301e\u0301i\u0301 A')
-        test('%(height)D', '1K')
-        test('%(height)5.2D', ' 1.08K')
+        test('%(height)D', '1kB')
+        test('%(height)5.2D', ' 1.08kB')
```

Comment:
this should only have K

Reply:
@pukkandan Actually this is an intentional lowercase `k`, I have linked the units table in the pull request, a lowercase `k` is used as the metric prefix, please see: [Kilobyte](https://en.wikipedia.org/wiki/Kilobyte) and [Multiple-byte units](https://en.wikipedia.org/wiki/Byte#Multiple-byte_units).

> In the International System of Units (SI) the prefix kilo means 1000 (103); therefore, one kilobyte is 1000 bytes. The unit symbol is kB.

> Definition of prefixes using powers of 10—in which 1 kilobyte (symbol kB) is defined to equal 1,000 bytes—is recommended by the International Electrotechnical Commission (IEC).

I'm not trying to be picky here, I was just using the standard units, the function `format_decimal_suffix` had an output with the units without the `B` so that's why I removed the `iB` from `format_bytes` and left the units to be decided by `format_decimal_suffix` as it takes a `factor` argument with `1000` or `1024`.

Reply:
I hadn't noticed the case change. I was saying that the `B` should not be added for these tests

If you can fix that, I can re-open this and merge the case change. But imo it isn't really important (Ive already pushed a fix for the `iB` issue)

Reply:
fixed in https://github.com/yt-dlp/yt-dlp/commit/abbeeebc4c20bc5c690d96c83e91e89effd04b81